components/wpt-results.html

```diff
@@ -194,6 +194,11 @@
             type: Boolean,
             computed: '_computeIsLatest(sha)'
           },
+          // ['browser@sha] spec for the runs to load
```

Comment:
Nit: Missing close `'`?

Reply:
Comment is a little terse. Maybe an example that includes multiple browsers and shas (and a description of what gets loaded).

Reply:
Done.